cmd/ocsp-updater/main.go

```diff
@@ -727,6 +760,7 @@ func main() {
 	dbURL, err := conf.DBConfig.URL()
 	cmd.FailOnError(err, "Couldn't load DB URL")
 	dbMap, err := sa.NewDbMap(dbURL, conf.DBConfig.MaxDBConns)
+	sa.SetSQLDebug(dbMap, auditlogger)
```

Comment:
It was. Now reverted. I do think we should probably be setting this for all of our SQL-using apps, and letting log level filtering remove it when we don't need it. But that's a separate PR.